src/api/goods.ts

```diff
@@ -1,11 +1,16 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
-  return fetch(API_URL)
-    .then(response => response.json());
+export async function getAll(): Promise<Good[]> {
+  return (await fetch(API_URL)).json();
```

Comment:
https://github.com/mate-academy/react_dynamic-list-of-goods/pull/472#pullrequestreview-758271121